app/helpers/application_helper.rb

```diff
@@ -1,2 +1,8 @@
 module ApplicationHelper
+  def markdown(text)
```

Comment:
I know this is what's taught in bloccit, but `markdown` isn't a very accurate method name here. It doesn't truly tell us what this method is doing, it's vague. 

We'd like intention-revealing names that explain the method outputs HTML from markdown. 

Some ideas to consider and come up with your own improved name possibly:

- `convert_markdown_to_html(text)`
- `markdown_to_html(markdown)`
- `to_html(markdown)`